agent/client/client.go

```diff
@@ -671,7 +671,7 @@ func (c *Client) handleStartJobRequest(p *agentpb.StartJobRequest) error {
 }
 
 func (c *Client) getMongoDSN(dsn string, files *agentpb.TextFiles, jobID string) (string, error) {
-	tempDir := filepath.Join(c.cfg.Get().Paths.TempDir, "mongodb-backup-restore", strings.Replace(jobID, "/", "_", -1))
+	tempDir := filepath.Join(c.cfg.Get().Paths.TempDir, "mongodb-backup-restore", strings.Replace(jobID, "/", "_", -1)) //nolint:gocritic
```

Comment:
What was this complaining about?

Reply:
It wants `string.ReplaceAll`

Reply:
That seems like a correct request.

Reply:
It is legit, problem is I can't predict the consequence.

Reply:
got it